<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgReforma</class>
 <widget class="QDialog" name="dlgReforma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lugar</string>
       </property>
       <property name="buddy">
        <cstring>txtLugar</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtLugar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarLugar">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Orden</string>
       </property>
       <property name="buddy">
        <cstring>txtOrden</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtOrden"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckMasculino">
       <property name="text">
        <string>Masculino</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="ckAbandonado">
       <property name="text">
        <string>Abandonado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Año</string>
       </property>
       <property name="buddy">
        <cstring>txtAbandonadoAno</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAbandonadoAno"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckRetomado">
       <property name="text">
        <string>Retomado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Año</string>
       </property>
       <property name="buddy">
        <cstring>txtRetomadoAno</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRetomadoAno"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btOtrosDatos">
       <property name="text">
        <string>&amp;Otros datos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtLugar</tabstop>
  <tabstop>btQuitarLugar</tabstop>
  <tabstop>txtOrden</tabstop>
  <tabstop>ckMasculino</tabstop>
  <tabstop>ckAbandonado</tabstop>
  <tabstop>txtAbandonadoAno</tabstop>
  <tabstop>ckRetomado</tabstop>
  <tabstop>txtRetomadoAno</tabstop>
  <tabstop>btOtrosDatos</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
